### ISSUE TITLE
Ugly icon

### ISSUE THREAD

**Issue by Karmacom** [Entware/entware#107]
Sorry for the offence, but the current entware version of minidlna is the most stable version and for many the only working version without memory leaks/cpu-hogging crash bugs, and it's horrific to have to have that penguin (made by linus in the 80's) staring at you. It has nothing to do with DLNA, movies or TVs, it just "hints" of the system its been compiled/running from, which is 100% uninteresting for 99% of the end-users.

Please, please make a ipk-package of the actual current (leaving everything else untouched exept the icons.h/icons.c) where it doesnt use that penguin as an icon. Any "neutral" icon, a TV, a mapfolder, or the icons.h/icons.c used in RMerlins firmwares for asuswrt (avaivable here on github) could be used.
I have tried to learn how to make ipk-packages from src, and tried to compile from src on my router (n66u) ,but "./configure" stops with errors of missing c compiler (basically cc or gcc, but dont know how to install that on n66u.

Thanks in advance if someone has the patience, interest and time to do this.

**Comment by ryzhovau** [comment]
I think it's better to send such issue to minidlna authors at http://sourceforge.net/projects/minidlna/

**Comment by Karmacom** [comment]
Please, I allready have with no response. All I want is a bugfree and
penguin-free minidlna for my n66u router, and this is the only way. I will
gladly pay you to compile a ipk-package of entware minidlna without the
penguin icon, how much would be fair for your time?

50USD? more?

I know alot of Asuswrt users with many minidlna problems would be gratefull
with a entware-version without that horrific logo (see, the problem is that
logo / penguin turns up on every DLNA-device connected to the router, the
TV's, tablets, phones etc.)



2015-01-12 8:06 GMT+01:00 Alexander Ryzhov <notifications@github.com>:

> I think it's better to send such issue to minidlna authors at
> http://sourceforge.net/projects/minidlna/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Entware/entware/issues/107#issuecomment-69535158>.
>



-- 
Mvh.

Magnus Monkerud
magnus@monkerud.com

**Comment by Karmacom** [comment]
Please, I allready have with no response. All I want is a bugfree and
penguin-free minidlna for my n66u router, and this is the only way. I will
gladly pay you to compile a ipk-package of entware minidlna without the
penguin icon, how much would be fair for your time?

50USD? more?

I know alot of Asuswrt users with many minidlna problems would be gratefull
with a entware-version without that horrific logo (see, the problem is that
logo / penguin turns up on every DLNA-device connected to the router, the
TV's, tablets, phones etc.)


2015-01-12 8:06 GMT+01:00 Alexander Ryzhov <notifications@github.com>:

> Closed #107 <https://github.com/Entware/entware/issues/107>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Entware/entware/issues/107#event-217226717>.
>



-- 
Mvh.

Magnus Monkerud
magnus@monkerud.com

**Comment by ryzhovau** [comment]
By respecting other users, I can't change program icon in repository. Some users (potentially) will be surprised, I imagine how they open another issue to bring old logo back because they love Linus pinguins:) What should I tell them? 

I can make ipk-package with a changed icons, it will be usable with Entware, but it will not be a part of Entware repo. Is it worth 50USD?

**Comment by Karmacom** [comment]
It is definately worth 50USD, I will gladly pay it, tell me where to put it
/ how and I will do it in advance.

I understand and respect all penguin-lovers, and that it will not be part
of the "official" repository, but it will be 100% identical to the
minidlna-entware-package that lies there now right? (exept for the icon
offcourse).

It needs to be the exact version / compilation that lies in the repository
now (identical to the version you get when typing "opkg install minidlna",
not an older/newer one, not any other "fixes" or patches", simply a 100%
identical copy, just with another icon instead (preferably the icon used by
Rmerlins bundled minidlna binary (can be found in his github for the
asuswrt build he makes, its probarbly an icon asus has put in there - it
looks like a router with three antennas).

Basically it would be wonderfull if i could just type "opkg install
http://someurl.com/special-minidlna-for-karmacom.ipk", or even just the
entware-version of the binary itself downloadable, so that i could just
install the normal / official entware-minidlna, then just copy that binary
into /opt/bin replacing the original entware-minidlna.

Sorry for typing so much, and again thanks in advance, you're saving my
OCD/autistic daily nightmare.



2015-01-12 12:46 GMT+01:00 Alexander Ryzhov <notifications@github.com>:

**Comment by ryzhovau** [comment]
> but it will be 100% identical to the minidlna-entware-package that lies there now right? (exept for the icon offcourse).

Right!

> It needs to be the exact version / compilation that lies in the repository now (identical to the version you get when typing "opkg install minidlna", not an older/newer one, not any other "fixes" or patches", simply a 100% identical copy, just with another icon instead (preferably the icon used by Rmerlins bundled minidlna binary (can be found in his github for the asuswrt build he makes, its probarbly an icon asus has put in there - it looks like a router with three antennas).

Here we go:
```
cd /opt/tmp
wget http://entware.wl500g.info/binaries/entware/archive/minidlna_1.1.4-1b_entware.ipk
opkg install ./minidlna_1.1.4-1b_entware.ipk
opkg flag hold minidlna
```

Please, provide a feedback.

**Comment by Karmacom** [comment]
Wow, that was fast, wonderfull!, I will test it over night / til tomorrow
to see if its as stable as the original one nad report back. Please provide
method of payment so I can pay you regardless right now. I pay for the
goodwill/try/effort, not for the result. (I believe in karma, paying only
for good results is bad karma).
If there's personal details regarding payment, just PM the details to me
here on my github-user.

Again, thank you, a million times.

2015-01-12 13:37 GMT+01:00 Alexander Ryzhov <notifications@github.com>:

> but it will be 100% identical to the minidlna-entware-package that lies
> there now right? (exept for the icon offcourse).
>
> Right!
>
> It needs to be the exact version / compilation that lies in the repository
> now (identical to the version you get when typing "opkg install minidlna",
> not an older/newer one, not any other "fixes" or patches", simply a 100%
> identical copy, just with another icon instead (preferably the icon used by
> Rmerlins bundled minidlna binary (can be found in his github for the
> asuswrt build he makes, its probarbly an icon asus has put in there - it
> looks like a router with three antennas).
>
> Here we go:
>
> cd /opt/tmp
> wget http://entware.wl500g.info/binaries/entware/archive/minidlna_1.1.4-1b_entware.ipk
> opkg install ./minidlna_1.1.4-1b_entware.ipk
> opkg flag hold minidlna
>
> Please, provide a feedback.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Entware/entware/issues/107#issuecomment-69563855>.
>



-- 
Mvh.

Magnus Monkerud
magnus@monkerud.com

**Comment by ryzhovau** [comment]
Ok then. My PayPal ID is `paypal@ryzhov-al.ru`.

**Comment by Karmacom** [comment]
Do i choose transfer -> family / friends or transfer -> purchase/commodity?
Dont you have a donata-page?

**Comment by ryzhovau** [comment]
> Do i choose transfer -> family / friends or transfer -> purchase/commodity?
family/friends.

> Dont you have a donata-page?
No. The most valuable contribution comes from those guys, who's spending their personal time to improve Entware.

**Comment by Karmacom** [comment]
Done! Sent you 60USD now. If you want me to pay if forward, contribute, if you tell me in simple steps how to setup a compilation enviroment for entware on n66u (or mimicing the n66u, or is entware "general" linux / debian openwrt buildroot?) then I can start making custom/forks for those in need, and contribute that way.

Thanks a billion times again.

**Comment by ryzhovau** [comment]
Take a look at https://github.com/Entware/entware/wiki/How-to-add-a-new-package

**Comment by Karmacom** [comment]
but how does the entware / enviroment i run in a linux-debian know what build/type i want to build it for? thats the part I dont understand. or is "entware" always only mipsel?, from there all i need is to make the openwrt buildroot / tools, then i can download gits/src's of entware packages here, modify them and compile and it will be exactly like the ipkg-packages officially released?

Am I asking strange questions then let me know (im used to perl,php, ahk, as, js, java building / compiling on Windows platform for only two arcitectures, 64bit or x86, so im new to this).

**Comment by ryzhovau** [comment]
> How does the entware / enviroment i run in a linux-debian know what build/type i want to build it for? thats the part I dont understand. or is "entware" always only mipsel?, from there all i need is to make the openwrt buildroot / tools, then i can download gits/src's of entware packages here, modify them and compile and it will be exactly like the ipkg-packages officially released?

MIPSEL binaries will be produced by toolchain: a set of tools like linker, compiler and so on, which will be downloaded (or compiled) to `/opt/entware` folder.

The whole logics of making packages (such as tracking dependencies, applying patches to sources and so on) is provided by OpenWrt buildroot (at `openwrt_trunk` folder).

So, if you want to modify `minidlna`, go to `openwrt_trunk/feeds/package/multimedia/minidlna` folder, make some changes and run `make package/minidlna/compile V=s` from `openwrt_trunk` folder. If all went well, you'll find new ipk-file at `openwrt_trunk/bin/entware/packages` folder. 

To learn more, please refer to OpenWrt Buildroot documentation.

**Comment by Karmacom** [comment]
Ok, great, I will try to build myself an enviroment in a VM with linux debian and follow the instructions from there. And again thank you for having patience with me on this,

Please let me know if the money don't show up.

**Comment by TeHashX** [comment]
I tried to compile minidlna and other packages but I get always same error on clean Ubuntu and Debian:

/opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc: 2: /opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc: Syntax error: "(" unexpected
make[3]: *** [blocksort.o] Error 2
make[3]: Leaving directory `/home/thx/openwrt_trunk/build_dir/target-mipsel-linux-gnu/bzip2-1.0.6'
make[2]: *** [/home/thx/openwrt_trunk/build_dir/target-mipsel-linux-gnu/bzip2-1.0.6/.built] Error 2
make[2]: Leaving directory `/home/thx/openwrt_trunk/feeds/packages/utils/bzip2'
make[1]: *** [package/feeds/packages/bzip2/compile] Error 2
make[1]: Leaving directory `/home/thx/openwrt_trunk'
make: *** [package/minidlna/compile] Error 2

thx@debian:~/openwrt_trunk$ which gcc
/usr/bin/gcc

thx@debian:~/openwrt_trunk$ /opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc -v
bash: /opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc: cannot execute binary file

thx@debian:~/openwrt_trunk$ sudo /opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc -v
[sudo] password for thx: 
/opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc: 2: /opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc: Syntax error: "(" unexpected
thx@debian:~/openwrt_trunk$

What do I miss?

**Comment by Karmacom** [comment]
What is your commandline? / batch / script you use to compile? Are you sure you embrace with proper language format according to c? (e.g, sometimes "(" is used in *nix-batch/sh scripts, while in java, php, c etc its { instead of ( etc. Often same condition-routines, but different dialect.

**Comment by TeHashX** [comment]
make package/minidlna/compile V=s

**Comment by ryzhovau** [comment]
> thx@debian:~/openwrt_trunk$ sudo /opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc -v
[sudo] password for thx:
/opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc: 2: /opt/entware/toolchain-entware/bin/mipsel-linux-uclibc-gcc: Syntax error: "(" unexpected

Let me guess. Are you on Debian x32?

**Comment by TeHashX** [comment]
Yes

**Comment by ryzhovau** [comment]
By default, an amd64 toolchain will be downloaded. Please:
* run `make clenall`,
* uncomment `FORCE_COMPILE=y` string in `config.mk` file,
* run `make` again.

**Comment by Karmacom** [comment]
"Let me guess. Are you on Debian x32?" <- is it because of hex/binary conversion functions not working properly on little endian vs big endian?

Sorry to snoop in, just curious, and have had similar problems before with binary conversion functions in php and perl when operating in 64-bit enviroments vs 32bit.

**Comment by TeHashX** [comment]
I didn't pay attention to 64bit toolchain and now compilation of a 32bit bit is on the go but very slow because I'm on wmvare player.

**Comment by Karmacom** [comment]
Ok, preliminery feedback: The minidlna seems very stable so far. Low cpu usage while streaming, stresstested with heavy samba-copying to/from media folder while running, and heavy transmission downloading directly to media-dir, minidlna updates and scans fine while doing it all so far, no crashes.

Before (using the inbuilt / bundles minidlna that comes with the FW) it used to crash every time i copied something to/from the media-dir and even non-indexed dirs aswell, and it forced transmission and itself to use 100%  cpu most of the time, even when idling it used 60-70%.

Now it uses below 1% when streaming, i hope it stays this way, its amazing. I love the penguin not being there, I can sleep without nightmares now.

**Comment by Karmacom** [comment]
Day two: Still running stable, no issues what so ever. Shrek is love.

**Comment by ryzhovau** [comment]
> "Let me guess. Are you on Debian x32?" <- is it because of hex/binary conversion functions not working properly on little endian vs big endian?

No. It's because x64 toolchain binaries was downloaded for x32 system.

> I didn't pay attention to 64bit toolchain and now compilation of a 32bit bit is on the go but very slow because I'm on wmvare player.

I've fixed that. 302bddd7566a6f4fd71ac6119abd5ee537771a88. From now on, appropriate toolchain will be chosen to download for x32/x64 hos systems.

**Comment by TeHashX** [comment]
Thanks

**Comment by Karmacom** [comment]
This thing is so stable I'm going take it all out and run two minidlna's simultaniously with different UUID's.

**Comment by Karmacom** [comment]
Day 3(4) still running stable. I tried running multiple instances, couldnt get it working, it doesnt like UUID= in its conf file, so skipped that and went with -u commandlineparameter instead, but didnt respond to that either, refusing to fire up a second instance.

**Comment by Karmacom** [comment]
Over a week now. Rock bottom solid this is.
Again - thank you!

**Comment by MelissaJS** [comment]
Hi guys! 

I recently got a RT-AC66U router, I understand it is a mipsel router. I agree with Karmacom, the bundled apps for streaming and downloading are buggy and unstable, so I googled for alternatives and found TeHashX's excellent compilations (transmission and debian-minidlna). 

I really struggled to install them, since I'm a total newbie to these matters; I had to learn about linux and how to use my mac's terminal LOL! However the instructions at www.hqt.ro are quite understandable. I love the fact that, having only "old" samsung TVs at my home, TeHashX's minidlna for debian (1.1.4) preserved bookmarking and added some amazing thumbnails and filmstrip to the files when navigating through my media.

I recently attached a second hard drive to the router and it's been like a nightmare since then. I have not been able to mount both into debian, I have tried several modifications to the files in the init.d folder, to no avail; I've given up. Then I recalled that transmission runs directly on entware, so I googled for an entware minidlna compilation and found you guys :)

I installed the most recent entware minidlna version (1.1.4-2) and is very very stable! However bookmarking is gone in my TVs :(

I also tried ryzhovau's penguinless version and is also great, but bookmarking is gone as well :( 
I understand this has nothing to do with entware, as it is something related to a flag that was removed since the 1.1.3 update to minidlna.

TeHashX, by any chance can you make an entware minidlna compilation based on the latest version, but preserving bookmarks for samsung TVs and including the amazing filmstrips? I'd give it a shot but I know nothing about compiling :(

I bet that with such a compilation, you could even add a tutorial for an entware minidlna installation at your hqt webpage!!

**Comment by ryzhovau** [comment]
@TeHashX?

**Comment by TeHashX** [comment]
I already tried in the past but as you know I'm not a developer, just using google a lot :)
Following packages are required in buildroot: libavutil-dev libexif-dev libjpeg8-dev libid3tag0-dev libFLAC-dev libvorbis-dev libsqlite3-dev libffmpegthumbnailer-dev, not sure if all but at least libffmpegthumbnailer-dev.
This is thumbnails patch http://sourceforge.net/p/minidlna/patches/_discuss/thread/28a9f79d/038b/attachment/video_thumbnail-1.1.4.patch

**Comment by MelissaJS** [comment]
Oh I see, same here hehe ;)

I guess I'll have to dig into building compilations :( And I don't even have a linux machine. Virtual box and debian run quite sluggish on my mac.

Is this the same patch you used to build the compilation at https://www.hqt.ro/minidlna-upnp-media-server-through-debian-wheezy-mipsel-optware/   ??

If so, then I assume it includes support for both thumbnails and bookmarking for samsung TVs. Am I right?

**Comment by TeHashX** [comment]
It's the same patch, just slightly different, bookmarks are enabled by default but thumbnails not.

**Comment by MelissaJS** [comment]
Great! thnx a lot TeHashX !

ryzhovau may I ask you to point me to a place where I can learn how to compile a fusion of your penguinless entware minidlna and the bookmark/thumbnail patch?

**Comment by MelissaJS** [comment]
@ryzhovau Do you have a git for minidlna - entware?

**Comment by ryzhovau** [comment]
I've ported `minidlna` package from [upstream](https://github.com/openwrt/packages) and sync with them once per month (or two). 

You can find every `minidlna` version ever been appeared in Entware [here](http://entware.wl500g.info/binaries/entware/archive/). 

That's all I can provide because I'm not ready to make some personal `minidlna` builds. If you want to compile you own version, please refer [this how-to](https://github.com/Entware/entware/wiki/How-to-add-a-new-package-or-rebuild-the-feed).

**Comment by MelissaJS** [comment]
Thanks for the answer @ryzhovau

Unfortunately the most recent versions of minidlna for entware do not include:
-Bookmarking for samsung TVs
-Support for video thumbnails
-Support for custom logo

I tried to learn some basics from your referred pages but this is proving to be rather difficult to me :(

I am in my openwrt_trunk directory, and I understand that the command:

sudo make package/minidlna/compile

is used to compile minidlna's latest version for entware. I do not know the URL where this is pulled from, but it is compiling a version that doesn't have the features I need.

How can I compile from a certain fork or git? What do I write in the command line?

Sorry I'm kinda lost here :(

**Comment by YasharF** [comment]
Hi @MelissaJS ,
>> I do not know the URL where this is pulled from
It is being pulled based on the URL and filename in the Makefile in package/minidlna

The version that is currently being pulled in, based on the OpenWRT repo, is specified in line 11 of https://github.com/openwrt/packages/blob/master/multimedia/minidlna/Makefile .  Note that at least PKG_MD5SUM line needs to be changed to correspond to the new file's MD5SUM. The URL for the file has a base which currently is specified on line 32 as "http://minidlna.sourceforge.net/"  If you go to that URL and then download, you should get a minidlna folder on sourceforge with all of the available versions including the one on line 11.  Now if you have a patched version of minidlna with the features that you want as a tarball or in a git repo, you would just need to make changes to your local repo to try it.  If you have a source for a fork in a different git repo, here is an example of a Makefile that uses a git source instead of a sourceforge tarball: https://github.com/openwrt/packages/blob/master/libs/hiredis/Makefile

Note that OpenWRT, applies a patch to the source as specified in https://github.com/openwrt/packages/tree/master/multimedia/minidlna/patches . Entware, also applies a few patches to the existing minidlna (see https://github.com/Entware/entware/blob/master/packages/minidlna/Makefile lines 11-14).  If you are changing the source repo, git may not be able to apply these patches as they are and you may need to modify the patches to make them work again in case if they are still needed.

If you manage to get the whole working, I would suggest submitting a patch (start from the folks in minidlna project) to incorporate the features in there.  After that, submit an update patch to OpenWRT, and eventually to Entware :)

Hope this helps

**Comment by alllexx88** [comment]
I've contacted @TeHashX to ask which patches were used to compile their debian minidlna versions, and it turns out it was latest [git](http://sourceforge.net/u/eastcheap/minidlna/ci/master/tree/) plus a [thumbnails patch](http://sourceforge.net/p/minidlna/patches/132/#bcb5). I then [ported](https://github.com/alllexx88/Optware-ng/blob/f834539b714cddb6b6916ebbb6a4fb27c4d509e6/make/minidlna.mk) it to Optware-ng. It is quite simple to port it to OpenWrt (and Entware) too, one additional step you need, apart from moving to git version and applying thumbnails patch, is porting the additional dependency `ffmpegthumbnailer`

**Comment by zyxmon** [comment]
I have switched from minidlna (compiled it for my NAS) to Serviio. But I want to share my experience with minidlna.
1. I use xbmc (kodi) to generate nfo files and thumbnails. These files are excepted by minidlna. Tese allows to search the film by name, not by filename and adds poser as a thumbnail.
2. As for bookmarking for samsung TVs - I have D series Samsungs and bookmarking always worked for them. The new models have problems with minidlna bookmarking. It was removed (temporary?) by the minidlna author. There is a patch for E series and it does not work for F.

**Comment by MelissaJS** [comment]
Thank you guys for your answers! 

@YasharF How can I find out the MD5SUM for my modified git? I managed to set up all the mods I needed here:  https://github.com/MelissaJS/minidlna.git

@alllexx88 Where exactly can I set the --enable-thumbnail switch? Is it in the command line while making the package? And how do I port the dependency ffmpegthumbnailer ?

@zyxmon I'll give a try to your xbmc trick once I manage to compile minidlna! Wouldn't it be so wonderful if the Readymedia guys managed to implement a method to directly show a movie poster pulled from a sort of online database, just like Plex does? :) The whole issue regarding bookmarking goes back to a very specific flag they removed since 1.1.3 and, as far as I understand from checking the official repo, it has been scheduled to put back this flag in the clients.c file for the next version. Sadly, they haven't released a new version for over a full year now :(

**Comment by YasharF** [comment]
@MelissaJS In case of a git repo as a source, I think the MD5SUM line is not needed.  See https://github.com/openwrt/packages/blob/master/libs/hiredis/Makefile as an example of the lines that are needed. See also http://wiki.openwrt.org/doc/devel/packages for more info on the various fields.

**Comment by MelissaJS** [comment]
Thank you @YasharF 

According to the reference you provided, I already modified the Makefile located at:

openwrt_trunk/feeds/packages/multimedia/minidlna/Makefile

These are the the code lines that I added/modified:

include $(TOPDIR)/rules.mk

PKG_NAME:=minidlna
PKG_VERSION:=1.1.4
PKG_RELEASE:=melissajs

PKG_SOURCE_PROTO:=git
PKG_SOURCE_URL:=https://github.com/MelissaJS/minidlna.git
PKG_SOURCE_VERSION:=f58dd249d6ed47a7e835463c3b04722972281dbb
PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz

PKG_LICENSE:=BSD-3-Clause
PKG_LICENSE_FILES:=COPYING
PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>

PKG_BUILD_PARALLEL:=0
PKG_BUILD_DEPENDS:=util-linux

PKG_INSTALL:=1

include $(INCLUDE_DIR)/package.mk
include $(INCLUDE_DIR)/nls.mk

define Package/minidlna
  SECTION:=multimedia
  CATEGORY:=Multimedia
  TITLE:=UPnP A/V & DLNA Media Server
  URL:=https://github.com/MelissaJS/minidlna.git
  DEPENDS:= +libpthread +libexif +libjpeg +libsqlite3 +libffmpeg \
        +libid3tag +libflac +libvorbis +libuuid +libbz2 \
        $(ICONV_DEPENDS) $(INTL_DEPENDS)
endef


While in the openwrt_trunk directory, I am running the command: sudo make package/minidlna/compile

This is resulting in the creation, without any errors, of the file minidlna_1.1.4-melissajs_entware.ipk
However, when I install this package in my router, it is not installing my repo! For some reason it is actually installing the original repo located at https://sourceforge.net/projects/minidlna/
(the installed minidlna.conf file is not the one I patched; it's the one from the original repo).

What am I doing wrong?

**Comment by YasharF** [comment]
@MelissaJS
I suspect you might be using <code>opkg install minidlna </code> command.  If that is the case, then you actually need to point opkg to use your test ipk file (aka minidlna_1.1.4-melissajs_entware.ipk).

1. SSH to the router and copy your test ipk file to a usb/temp location on the router
2. Remove the existing installed package (i.e. opkg remove minidlna, etc.)
3. Install the updated version from usb/temp (I think: opkg install /tmp/minidlna_1.1.4-melissajs_entware.ipk )

**Comment by MelissaJS** [comment]
@YasharF I updated the SHA as per your instructions and recompiled :)

I copied the new minidlna_1.1.4-melissajs_entware.ipk file to the /opt/tmp folder in the router and ran the following command: opkg install opt/tmp/minidlna_1.1.4-melissajs_entware.ipk
and i get the following:

Installing minidlna (1.1.4-melissajs) to root...
Configuring minidlna.

That is, apparently everything is ok. But then again, the installed minidlna version is the official one, not my custom version. Weird isn't it?

Is there something else I am missing? Is it a good idea to try from a sourceforge repo instead?
